Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/base_turret.yml

```diff
@@ -96,18 +110,68 @@
     proto: CartridgeCaselessRifle
     capacity: 500
 
+- type: entity # Magazine fed turret
+  parent: BaseWeaponTurretNF
+  id: BaseWeaponTurretMagazineFed
+  name: ballistic turret (magazine fed)
+  description: A ballistic machine gun auto-turret with replaceable magazine.
+  abstract: true
+  components:
+  - type: Gun
+    fireRate: 6
+    selectedMode: FullAuto
+    availableModes:
+    - FullAuto
+    soundGunshot: /Audio/Weapons/Guns/Gunshots/gun_sentry.ogg
+  - type: MagazineAmmoProvider
+    #boltClosed: null
```

Comment:
Is this inherited?  A test?

Reply:
Was left from earlier stages of testing. Gonna delete it.